sites/footers/old-footers-fr.html

```diff
@@ -7,13 +7,164 @@
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/immigration-citoyennete.html">Immigration et citoyenneté&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://voyage.gc.ca/">Voyage et tourisme&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/entreprises.html">Entreprises&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/prestations.html">Prestations&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/sante.html">Santé&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/impots.html">Impôts&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/environnement.html">Environnement et ressources naturelles&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/defense.html">Sécurité nationale et défense&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/culture.html">Culture, histoire et sport&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/police.html">Services de police, justice et urgences&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/transport.html">Transport et infrastructure&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.international.gc.ca/world-monde/index.aspx?lang=fra">Le Canada et le monde&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/finance.html">Argent et finances&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/science.html">Science et innovation&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/autochtones.html">Autochtones&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/veterans-militaire.html">Vétérans et militaires&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/services/jeunesse.html">Jeunesse&lt;/a>&lt;/li>
+				&lt;/ul>
+			&lt;/nav>
+		&lt;/div>
+	&lt;/div>
+
+	&lt;div class="gc-sub-footer">
+		&lt;div class="container d-flex align-items-center">
+			&lt;nav>
+				&lt;h3 class="wb-inv">Organisation du gouvernement du Canada&lt;/h3>
+				&lt;ul>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/sociaux.html">Médias sociaux&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/mobile.html">Applications mobiles&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/gouvernement/a-propos.html">À propos de Canada.ca&lt;/a>&lt;/li>&lt;li>&lt;a href="https://www.canada.ca/fr/transparence/avis.html">Avis&lt;/a>&lt;/li>
+					&lt;li>&lt;a href="https://www.canada.ca/fr/transparence/confidentialite.html">Confidentialité&lt;/a>&lt;/li>
+				&lt;/ul>
+			&lt;/nav>
+			&lt;div class="wtrmrk align-self-end">
+				&lt;img src="/dist/GCWeb/assets/wmms-blk.svg" alt="Symbole du gouvernement du Canada">
+			&lt;/div>
+		&lt;/div>
+	&lt;/div>
+&lt;/footer></code></pre>
+</details>
```

Comment:
This is not needed. This page lists the previous versions of the footer; not the current version.
```suggestion
```